src/pages/future/index.tsx

```diff
@@ -0,0 +1,100 @@
+import Image from "next/image";
+import React from "react";
+import useSWR from "swr";
+
+import { UpcomingLaunches } from "@src/@types/graphql-schema";
+import { LoadingComponent } from "@src/components/loading-component/loading-component";
+import { TD } from "@src/components/table";
+import { UPCOMING_LAUNCHES_QUERY } from "@src/graphql/launches/queries";
+
+async function fetcher(
+  url: string,
+  query: string,
+): Promise<UpcomingLaunches[]> {
+  return fetch(url, {
+    method: "POST",
+    headers: {
+      "Content-type": "application/json",
+    },
+    body: JSON.stringify({ query }),
+  }).then(async (response) => {
+    const unmarshalledResponse = await response.json();
+
+    return unmarshalledResponse.data.upcomingLaunches;
+  });
+}
```

Comment:
I think we should decide which tool between simple `fetch` and `graphql-request`'s `request` we will use and uniformize it to use only one of the both ? (and extracting the fetcher logic to avoid duplication around the codebase). Wdyt ?

Reply:
My thought on this particular fetcher was that, as I don't need any variables to be passed through, I use the most simple tool.

I agree that we should uniformed our fetcher there. My question is, do you think we should use `graphql-request`, or do we pass the variables in the body of the request? I'm not sure it adds anything we could do easily ourselves. What do you think?

Reply:
We will try with the original fetch 👍